akvo/rest/views/project_update.py

```diff
@@ -40,30 +38,29 @@ def get_queryset(self):
         We don't use the default filter_fields, because Up filters on
         datetime for last_modified_at, and they only support a date, not datetime.
         """
-        queryset = self.queryset
         created_at__gt = self.request.QUERY_PARAMS.get('created_at__gt', None)
         if created_at__gt is not None:
-            queryset = queryset.filter(created_at__gt=created_at__gt)
+            self.queryset = self.queryset.filter(created_at__gt=created_at__gt)
         created_at__lt = self.request.QUERY_PARAMS.get('created_at__lt', None)
         if created_at__lt is not None:
-            queryset = queryset.filter(created_at__lt=created_at__lt)
+            self.queryset = self.queryset.filter(created_at__lt=created_at__lt)
         last_modified_at__gt = self.request.QUERY_PARAMS.get('last_modified_at__gt', None)
         if last_modified_at__gt is not None:
-            queryset = queryset.filter(last_modified_at__gt=last_modified_at__gt)
+            self.queryset = self.queryset.filter(last_modified_at__gt=last_modified_at__gt)
         last_modified_at__lt = self.request.QUERY_PARAMS.get('last_modified_at__lt', None)
         if last_modified_at__lt is not None:
-            queryset = queryset.filter(last_modified_at__lt=last_modified_at__lt)
+            self.queryset = self.queryset.filter(last_modified_at__lt=last_modified_at__lt)
         # Get updates per organisation
         project__partners = self.request.QUERY_PARAMS.get('project__partners', None)
         if project__partners:
-            queryset = queryset.filter(project__partners=project__partners)
+            self.queryset = self.queryset.filter(project__partners=project__partners)
         user__organisations = self.request.QUERY_PARAMS.get('user__organisations', None)
         if user__organisations:
-            queryset = queryset.filter(user__organisations=user__organisations)
-        return queryset
+            self.queryset = self.queryset.filter(user__organisations=user__organisations)
+        return super(ProjectUpdateViewSet, self).get_queryset()
```

Comment:
I think a few lines can be removed here (NOTE: untested code):

```python
    def get_queryset(self):
        """
        Allow simple filtering on selected fields.
        We don't use the default filter_fields, because Up filters on
        datetime for last_modified_at, and they only support a date, not datetime.
        """
        query_keys = [
            'created_at__gt',
            'created_at__lt',
            'last_modified_at__gt',
            'last_modified_at__lt',
            'project__partners',
            'user__organisations',
        ]

        for key in query_keys:
            value = self.request.QUERY_PARAMS.get(key, None)
            if value:
                self.queryset = self.queryset.filter(**{key: value})

        return super(ProjectUpdateViewSet, self).get_queryset()
```